Source/Urho3D/Core/Variant.h

```diff
@@ -174,7 +174,7 @@ struct URHO3D_API ResourceRefList
     /// Return hash value for HashSet & HashMap.
     unsigned ToHash() const
     {
-        return type_.ToHash() * 31 + ea::hash<StringVector>()(names_);
+        return type_.ToHash() * 31 + static_cast<unsigned>(ea::hash<StringVector>()(names_));
```

Comment:
@eugeneko what do you think, considering 64bit systems?

Reply:
This produces warning on arm64 and x64 builds with clang for iOS. Along with many other similar code lines.